fibers/web/server.scm

```diff
@@ -249,7 +249,8 @@ on the procedure being called at any particular time."
                                (inet-pton family host)
                                INADDR_LOOPBACK))
                      (port 8080)
-                     (socket (make-default-socket family addr port)))
+                     (socket (make-default-socket family addr port))
+                     (init-thunk (lambda () #nil)))
```

Comment:
IIRC #nil is for Emacs lisp compatibility and its better to use #f or '() in scheme.

> ‘nil’ in ELisp is an amalgam of Scheme’s ‘#f’ and ‘'()’.  It is false,
and it is the end-of-list; thus it is a boolean, and a list as well.
>
> Guile has chosen to support ‘nil’ as a separate value, distinct from
‘#f’ and ‘'()’.  This allows existing Scheme and Elisp code to maintain
their current semantics.  ‘nil’, which in Elisp would just be written
and read as ‘nil’, in Scheme has the external representation ‘#nil’.

– (guile) Nil

Reply:
I’m don’t have anything to do with fibers though so don’t listen to me!